docs/reference/commandline/volume_create.md

```diff
@@ -0,0 +1,41 @@
+<!--[metadata]>
++++
+title = "volume create"
+description = "The volume create command description and usage"
+keywords = ["volume, create"]
+[menu.main]
+parent = "smn_cli"
+weight=1
++++
+<![end-metadata]-->
+
+# volume create
```

Comment:
<!--[metadata]>
+++
title = "volume create"
description = "The volume create command description and usage"
keywords = ["volume, create"]
[menu.main]
parent = "smn_cli"
+++
<![end-metadata]-->

# volume create

    Usage: docker volume create [OPTIONS]

    Create a volume

    -d, --driver=local    Specify volume driver name
    --help=false          Print usage
    --name=               Specify volume name
    -o, --opt=map[]       Set driver specific options

Creates a new volume that containers can can consume and store data in. If a name is not specified, Docker generates a random name. You create a volume and then configure the container to use it, for example:

  $ docker volume create --name hello
  $ docker run -d -v hello:/world busybox ls /world

Multiple containers can use the same volume in the same time period.

## Driver specific options

Some volume drivers may take options to customize the volume creation. Use the `-o` or `--opt` flags to pass driver options:

  $ docker volume create --opt size=10G

These options are passed directly to the volume driver. Options for
different volume drivers may do different things (or nothing at all).

*Note*: The built-in volume driver does not currently accept any options.